feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file is also added locally using Visual Studio in the feature-1 branch.
+
```

Comment:
It's really good that you've added this line of code in here.

Reply:
Thanks for this comment, Ozlem!